force-app/main/default/classes/CON_ContactMerge_TEST.cls

```diff
@@ -782,18 +783,44 @@ public class CON_ContactMerge_TEST {
 
         selectMasterRecord(controller, con.id);
 
-        //merge them
+        // Act: WHEN one Individual Bucket Contact is merged into another Bucket Contact
```

Comment:
Do you think this should be wrapped in a Test.Start/Stop?

Reply:
I'll add it.  I don't think it hurts, but I also don't think it is totally necessary since the offending behavior is observable (with the bug in place) without it.  It would be valuable if there was a specific use case or configuration where we _knew_ that the offending operation ran async.

Reply:
Good point about the issue being reproducible in this without start/stop.